Library/Homebrew/sandbox.rb

```diff
@@ -63,7 +63,7 @@ def allow_write_cellar(formula)
 
   # Xcode projects expect access to certain cache/archive dirs.
   def allow_write_xcode
-    allow_write_path "/Users/#{ENV["USER"]}/Library/Developer"
+    allow_write_path "#{ENV["HOME"]}/Library/Developer"
```

Comment:
Are you sure we haven't already overrides `HOME` when this is used? We do so in `formula.rb` in a few places.

Reply:
@MikeMcQuaid Very good point.

I’ll consider obtaining the path from Directory Services directly.

Reply:
I think the sandbox is set up before a formula is staged but I haven't audited all possible call paths.